app/javascript/chat/content.jsx

```diff
@@ -39,26 +39,22 @@ export class Content extends Component {
   };
 
   render() {
-    const {
-      onTriggerContent,
-      fullscreen,
-      resource,
-      closeReportAbuseForm,
-    } = this.props;
+    const { onTriggerContent, fullscreen, resource, closeReportAbuseForm } =
+      this.props;
     if (!resource) {
       return '';
     }
 
     return (
       // TODO: A button (role="button") cannot contain other interactive elements, i.e. buttons.
       // TODO: These should have key click events as well.
+      // eslint-disable-next-line jsx-a11y/click-events-have-key-events
       <div
         className="activechatchannel__activecontent activechatchannel__activecontent--sidecar"
         id="chat_activecontent"
         onClick={onTriggerContent}
         role="button"
         tabIndex="0"
-        aria-hidden="true"
```

Comment:
I've removed this, as the cypress tests highlighted we'd effectively blocked assistive tech users from this feature - I'm assuming it had been added instead of the `eslint-disable` so we'd pass the lint step despite the TODOs